travelnet.lua

```diff
@@ -81,7 +81,8 @@ minetest.register_node("travelnet:travelnet", {
     on_place = function(itemstack, placer, pointed_thing)
 
        local pos = pointed_thing.above;
-       if( minetest.get_node({x=pos.x, y=pos.y+1, z=pos.z}).name ~= "air" ) then
+       local node_name = minetest.get_node({x=pos.x, y=pos.y+1, z=pos.z}).name
+       if( not node_name or not minetest.registered_nodes[node_name].buildable_to ) then
```

Comment:
```suggestion
       local def = minetest.registered_nodes[
             minetest.get_node({x=pos.x, y=pos.y+1, z=pos.z}).name]
       if not def or not def.buildable_to then
```
node_name is always given, you need to test if the node definition exists. For unknown nodes, where a node named node_name is not registered, the defintion does not exist.